modules/history/assets/js/history/component.js

```diff
@@ -7,9 +7,9 @@ export default class Component extends ComponentBase {
 
 	defaultCommands() {
 		return {
-			do: ( args ) => $e.run( 'document/history/do', args ),
-			undo: () => $e.run( 'document/history/undo' ),
-			redo: () => $e.run( 'document/history/redo' ),
+			do: ( args ) => $e.createCommand( 'document/history/do', args ),
```

Comment:
Why not import?

Reply:
Fixed here: https://github.com/elementor/elementor/pull/10868